### ISSUE TITLE
Language-Agnostic SEntence Representations

### ISSUE THREAD

**Issue by alanakbik** [zalandoresearch/flair#427]
Facebook just open sourced multilingual [sentence embeddings](https://github.com/facebookresearch/LASER). 

Has anyone tried then out yet? Perhaps we should integrate them into Flair?

**Comment by MahmoudWahdan** [comment]
@alanakbik Good Idea. It should be integrated!

**Comment by stefan-it** [comment]
One problem is, that *LASER* is not yet a python library and the other problem is that some dependencies are non-python tools like `fastBPE`, that needs to be compiled first with the `install_external_tools.sh` or the Moses tokenization scripts. Thus, I think a lot of work needs to be done to easily use the language agnostic sentence representations in `flair`.

**Comment by stefan-it** [comment]
Btw: another nice paper from Facebooks Research is *Cross-lingual Language Model Pretraining* from [here](https://arxiv.org/abs/1901.07291). They showed the training of three different cross-lingual language model approaches like: causal languagel modeling, masked language modeling (slightly modified variant of the masked language modeling in the BERT paper) and translation language modeling. Results on XNLI are better than the one in the LASER paper. But unfortunately the code has not been released yet.

**Comment by alanakbik** [comment]
Ah that's too bad - from their [post](https://code.fb.com/ai-research/laser-multilingual-sentence-embeddings/) it seemed like LASER would be PyTorch, but you're right there seem to be lots of external dependencies, non-python stuff that needs to be installed, environment variables set etc.

**Comment by oterrier** [comment]
Hi
The new release 1.1.2 of [LASER](https://github.com/yannvgn/laserembeddings) has removed the dependencies to non python packages so maybe we could reconsider its integration ?
What do you think?

Best regards
Olivier Terrier
